### PR TITLE
Update pgo data versions, and remove excess nuget details handling old ibc data

### PR DESCRIPTION
**Pull request by davidwrighton**

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -122,14 +122,10 @@
...
-    <optimizationlinuxx64MIBCRuntimeVersion>99.99.99-master-20210415.12</optimizationlinuxx64MIBCRuntimeVersion>
-    <optimizationwindows_ntx64IBCCoreFxVersion>99.99.99-master-20200806.6</optimizationwindows_ntx64IBCCoreFxVersion>
-    <optimizationlinuxx64IBCCoreFxVersion>99.99.99-master-20200806.6</optimizationlinuxx64IBCCoreFxVersion>
-    <optimizationwindows_ntx64IBCCoreCLRVersion>99.99.99-master-20200806.6</optimizationwindows_ntx64IBCCoreCLRVersion>
-    <optimizationlinuxx64IBCCoreCLRVersion>99.99.99-master-20200806.6</optimizationlinuxx64IBCCoreCLRVersion>
-    <optimizationPGOCoreCLRVersion>99.99.99-master-20210420.2</optimizationPGOCoreCLRVersion>
+    <optimizationwindows_ntx64MIBCRuntimeVersion>1.0.0-prerelease.21228.9</optimizationwindows_ntx64MIBCRuntimeVersion>
+    <optimizationwindows_ntx86MIBCRuntimeVersion>1.0.0-prerelease.21228.9</optimizationwindows_ntx86MIBCRuntimeVersion>
+    <optimizationlinuxx64MIBCRuntimeVersion>1.0.0-prerelease.21228.9</optimizationlinuxx64MIBCRuntimeVersion>
+    <optimizationPGOCoreCLRVersion>1.0.0-prerelease.21228.9</optimizationPGOCoreCLRVersion>
```
Comment (epananth) [inline eng/Versions.props:128]: what about the PGO packages?

**Comment by epananth** [comment]
I was thinking, I should get the PR merged in optimization repo and then get the latest package versions with the latest sha and then update this PR..

File: eng/Versions.props (hunk 1 above)
Comment (davidwrighton) [reply eng/Versions.props:128]: optimizationPGOCoreCLRVersion refers to a metapackage which contains the references to the arch specific ones.

**Comment by davidwrighton** [comment]
@epananth  I'd rather not wait, as the goal I have with this PR is to get the repo into a state where the automated update will perform an actual update. I specifically would prefer to have slightly old data here, so that it actually does something when the update comes around.

File: eng/Versions.props (hunk 1 above)
Comment (epananth) [reply eng/Versions.props:128]: great

**Review by epananth — APPROVED** [review]

**Pull request merged**